src/ctl.c

```diff
@@ -920,6 +924,15 @@ static const ctl_named_node_t experimental_node[] = {
 	{NAME("thread"),	CHILD(named, experimental_thread)}
 };
 
+static const ctl_named_node_t malloc_conf_node[] = {
+	{NAME("file"),		CTL(malloc_conf_file)},
+	{NAME("env_var"),	CTL(malloc_conf_env_var)},
+	{NAME("global_var_malloc_conf"),
```

Comment:
The `_malloc_conf` suffix seems redundant. Maybe `global_var` and `global_var_2_conf_harder`? So the path would look like `opt.malloc_conf.global_var`